cloud/services/object_storage_buckets.go

```diff
@@ -30,7 +46,7 @@ func EnsureObjectStorageBucket(ctx context.Context, bScope *scope.ObjectStorageB
 		Region:      bScope.Bucket.Spec.Region,
 		Label:       bScope.Bucket.Name,
 		ACL:         linodego.ObjectStorageACL(bScope.Bucket.Spec.ACL),
-		CorsEnabled: bScope.Bucket.Spec.CorsEnabled,
+		CorsEnabled: util.Pointer(bScope.Bucket.Spec.CorsEnabled),
```

Comment:
we shouldn't need to use this util here
```suggestion
		CorsEnabled: &bScope.Bucket.Spec.CorsEnabled,
```